library/ssl_tls13_generic.c

```diff
@@ -1567,4 +1567,59 @@ int mbedtls_ssl_tls13_check_received_extension(
     return MBEDTLS_ERR_SSL_UNSUPPORTED_EXTENSION;
 }
 
+#if defined(MBEDTLS_SSL_RECORD_SIZE_LIMIT)
+/* From RFC 8449:
+ * The ExtensionData of the "record_size_limit" extension is
+ * RecordSizeLimit:
+ *     uint16 RecordSizeLimit;
+ */
+MBEDTLS_CHECK_RETURN_CRITICAL
+int mbedtls_ssl_tls13_parse_record_size_limit_ext(mbedtls_ssl_context *ssl,
+                                                  const unsigned char *buf,
+                                                  const unsigned char *end)
+{
+    const ptrdiff_t extension_data_len = end - buf;
+    if (extension_data_len != 2) {
```

Comment:
I would suggest extracting `2` and `64` as constants to `ssl_misc.h`, if `MBEDTLS_SSL_RECORD_SIZE_LIMIT` is defined.

Reply:
done